cmd/db/db.go

```diff
@@ -203,8 +203,19 @@ func (c *DatabaseCommand) adminMigrationsCmd(ctx context.Context, globalOptions
 	}
 
 	executeMigrationsFn := func(ctx context.Context, dir migrate.MigrationDirection, count int) error {
-		if err := ExecuteMigrations(ctx, globalOptions.DatabaseURL, dir, count, migrations.AdminMigrationRouter); err != nil {
-			return fmt.Errorf("executing migrations for %s: %w", adminCmd.Name(), err)
+		dbURL, err := router.GetDNSForAdmin(globalOptions.DatabaseURL)
```

Comment:
Not related to this PR, but I believe this should be switched to `GetDSNForAdmin()`